.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,266 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'mkdocs.yml'
+      - '**.md'
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    paths-ignore:
```

Comment:
Same as above:

 > IMO no need to ignore anything: github actions are fast enough to provide good feedback to everyone, and if we add markdown linting tasks in the future, this will just be in the way